package/MDAnalysis/coordinates/chain.py

```diff
@@ -56,13 +56,99 @@
 from . import core
 
 
+def multi_level_argsort(l):
+    """
+    This sorts based on the start time.
+
+    Parameters
+    ----------
+    l : list
```

Comment:
Can you add what `l` looks like and a simple example would be really helpful for anyone who comes after you and wants to understand the code.